meteor-app/imports/ui/member/member-list-in.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import { List, Header } from 'semantic-ui-react'
+import MemberCardIn from './member-card-in'
+
```

Comment:
Something rankles with the choice of name here. We got away from the mess of "checkedIn", "checkIn", "checkInList" etc, which was mighty confusing. We need a better choice of words, however, I think the real problem is that we are putting state into the component name.

In terms of component reusability, maybe we should just call it MemberCardSmall. It might be used in different places, and the state is still applicable (with some visual indicator). For example the list at the bottom of the page might show the people that have been here today, and the ones that are still here have a green border. We can discuss this further - ie design for reusability, but I hope you get the gist of where I am coming from for now